packages/components/data-table-manager/src/data-table-manager-provider/data-table-manager-provider.tsx

```diff
@@ -17,43 +14,8 @@ const DataTableManagerContext = createContext<TDataTableManagerContext>({
   isCondensed: true,
 });
 
-export const useDataTableManagerContext = () => {
-  const { columns, topBar, displaySettings, onSettingsChange, columnManager } =
-    useContext(DataTableManagerContext);
-
-  const areDisplaySettingsEnabled = Boolean(
-    displaySettings && !displaySettings.disableDisplaySettings
-  );
-  const isWrappingText =
-    areDisplaySettingsEnabled && displaySettings!.isWrappingText;
-
-  const contextValue = useMemo(
-    () => ({
-      columns: columns.map((column) => ({
-        ...column,
-        isTruncated: areDisplaySettingsEnabled
-          ? isWrappingText
-          : column.isTruncated,
-      })),
-      isCondensed: areDisplaySettingsEnabled && displaySettings!.isCondensed,
-      displaySettings,
-      topBar,
-      onSettingsChange,
-      columnManager,
-    }),
-    [
-      columns,
-      displaySettings,
-      topBar,
-      onSettingsChange,
-      columnManager,
-      areDisplaySettingsEnabled,
-      isWrappingText,
-    ]
-  );
-
-  return contextValue;
-};
+export const useDataTableManagerContext = () =>
+  useContext(DataTableManagerContext);
```

Comment:
It would be great to check for the context: if the context does not exist, we can throw an error explaining what's happening ([some reasoning about it](https://kentcdodds.com/blog/how-to-use-react-context-effectively#the-custom-consumer-hook)).

Reply:
Updated here: https://github.com/commercetools/ui-kit/pull/2807/commits/e04a5226ed7a7bde3743ec5f6a9f1d607e92b562

Thank you